raft/util.go

```diff
@@ -93,3 +93,18 @@ func DescribeEntry(e pb.Entry, f EntryFormatter) string {
 	}
 	return fmt.Sprintf("%d/%d %s %s", e.Term, e.Index, e.Type, formatted)
 }
+
+func limitSize(ents []pb.Entry, maxSize uint64) []pb.Entry {
```

Comment:
that is fine. see the comment on the interface. and we should not warn user... this is not a strict limiter.